README.md

```diff
@@ -1 +1,12 @@
-# IslamWalid-Container
\ No newline at end of file
+# Container
+Simple Container implementation in go using linux namespaces and cgroups.
+
+## Usage:
+```
+git clone https://github.com/codescalersinternships/IslamWalid-container.git
+cd IslamWalid-container
+mkdir rootfs
+mkdir rootfs/dev rootfs/bin rootfs/proc
+./busybox --install ./rootfs/bin
+sudo ./container run <command>
```

Comment:
I really think this should be `run <rootfs> <command>` because your app also need to know what directory to change root to before running the `command`